ext/intl/breakiterator/breakiterator_iterators.stub.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
Unfortuantely, I couldn't register `IntlPartsIterator` without extracting its stub to its own file, even though I tried multiple approaches.

Reply:
What was the problem with it?

Reply:
The problem was that originally, `breakiterator_arginfo.h` had to be included both in `breakiterator_class.cpp` and `breakiterator_iterators.cpp`, so that the class registering functions (e.g. `register_class_IntlPartsIterator`) became duplicate symbols.